sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-cosmos-multi-database-multi-account/src/main/java/com/azure/spring/sample/cosmos/multi/database/multiple/account/PrimaryDatasourceConfiguration.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.sample.cosmos.multi.database.multiple.account;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.spring.data.cosmos.config.AbstractCosmosConfiguration;
+import com.azure.spring.data.cosmos.config.CosmosConfig;
+import com.azure.spring.data.cosmos.core.ReactiveCosmosTemplate;
+import com.azure.spring.data.cosmos.core.convert.MappingCosmosConverter;
+import com.azure.spring.data.cosmos.repository.config.EnableReactiveCosmosRepositories;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class PrimaryDatasourceConfiguration {
+
+    private static final String PRIMARY_DATABASE = "primary_database";
+
+    @Bean
+    @Primary
+    @ConfigurationProperties(prefix = "azure.cosmos.primary")
+    public CosmosProperties primary() {
+        return new CosmosProperties();
+    }
+
+    @Bean
+    public CosmosClientBuilder primaryClientBuilder(@Qualifier("primary") CosmosProperties primaryProperties) {
+        return new CosmosClientBuilder()
+            .key(primaryProperties.getKey())
+            .endpoint(primaryProperties.getUri());
+    }
+
+    @EnableReactiveCosmosRepositories(basePackages = "com.azure.spring.sample.cosmos.multi.database.multiple.account.repository",
+        reactiveCosmosTemplateRef = "primaryDatabaseTemplate")
+    public class PrimaryDatabaseConfiguration extends AbstractCosmosConfiguration{
```

Comment:
Can we delete `PrimaryDatabaseConfiguration` and move related content(parent class, annotation, bean definition) to `PrimaryDatasourceConfiguration`?